node/pkg/devnet/hostname.go

```diff
@@ -10,6 +10,7 @@ import (
 
 // GetDevnetIndex returns the current host's devnet index (i.e. 0 for guardian-0).
 func GetDevnetIndex() (int, error) {
+	return 0, nil
```

Comment:
Is this leftover from testing? Maybe it should be deleted? Doesn't seem good to short-circuit this function.

Reply:
Ohhhh. Good catch. I thought I removed that change when I pushed. I'll fix that.